packages/protoplugin/src/ecmascript/names.ts

```diff
@@ -43,7 +43,7 @@ export function generateFilePath(
   ) {
     return wktFrom;
   }
-  return "./" + file.name + "_pbv2.js";
+  return "./" + file.name + "_pb.js";
```

Comment:
This must match the suffix.

Everything else is renaming files and updating imports.